README.md

```diff
@@ -338,18 +338,20 @@ Prints out most recent the _StackDriver logs_. These are logs from `console.log`
 - `--open`: Open StackDriver logs in a browser.
 - `--setup`: Setup StackDriver logs.
 - `--watch`: Retrieves the newest logs every 5 seconds.
+- `--simplified`: Removes timestamps from the logs.
 
 #### Examples
 
 ```text
 clasp logs
```

Comment:
We need to change this line if we are going to not show timestamps in the example.

```text
clasp logs --simplified
```

Reply:
I really didn't do my due diligence with the documentation 😥. Good catch.